packages/lablgtk3/lablgtk3.3.0.beta5/opam

```diff
@@ -17,11 +17,10 @@ doc: "https://garrigue.github.io/lablgtk/lablgtk3"
 
 depends: [
   "ocaml"     {         >= "4.05.0" & < "4.10" }
-  "dune"      {         >= "1.4.0"
-                      & != "1.7.0"
-                      & != "1.7.1"  } # Due to dune/dune#1833
+  "dune"      {         >= "1.8.0"  }
   "dune-configurator"
   "conf-gtk3" { build & >= "18"     }
+  "conf-gtksourceview3" { build     }
```

Comment:
I'm unsure why this is required. Could you elaborate?

Reply:
lablgtk also contains an OCaml wrapper for gtksourceview3. If this is not installed, it will fail. I am not so sure why I put in the "build", though - likely I just copied it from the conf-gtk3. Obviously both are required for build and run, unless one assumes that these packages just install the dev headers, which is not the case on Mac and Windows. On Linux the runtime is installed by default, so it would suffice to install the dev headers. Maybe this is why "build" is written here.

Is there a policy if a conf package for a library is the runtime or dev version?